packages/wallets/src/features/cashier/modules/Transfer/hooks/useTransferMessages/__tests__/useTransferMessages.spec.tsx

```diff
@@ -0,0 +1,204 @@
+        },
+        type: 'fiat' as const,
+    },
+    demo_account: 0 as const,
+    displayBalance: '10000.00 USD',
+    landingCompanyName: 'svg' as const,
+    loginid: '',
+    transfer: 'all',
+};
+
+const mockTrading = {
+    account_category: 'trading' as const,
+    account_type: 'ctrader' as const,
+    accountName: 'Deriv cTrader',
+    balance: '0.00',
+    currency: 'USD',
+    currencyConfig: {
+        code: 'USD',
+        display_code: 'USD',
+        fractional_digits: 1,
+        is_AUD: false,
+        is_BTC: false,
+        is_BUSD: false,
+        is_crypto: false,
+        is_DAI: false,
+        is_deposit_suspended: 0 as const,
+        is_ETH: false,
+        is_EUR: false,
+        is_EURS: false,
+        is_eUSDT: false,
+        is_fiat: true,
+        is_GBP: false,
+        is_IDK: false,
+        is_LTC: false,
+        is_PAX: false,
+        is_suspended: 0 as const,
+        is_TUSD: false,
+        is_tUSDT: false,
+        is_USB: false,
+        is_USD: true,
+        is_USDC: false,
+        is_USDK: false,
+        is_USDT: false,
+        is_withdrawal_suspended: 0 as const,
+        name: 'US Dollar',
+        stake_default: 0.1,
+        transfer_between_accounts: {
+            fees: {},
+            limits: { max: 0.1, min: 0.1 },
+            limits_ctrader: { max: 0.1, min: 0.1 },
+            limits_derivez: { max: 0.1, min: 0.1 },
+            limits_dxtrade: { max: 0.1, min: 0.1 },
+            limits_mt5: { max: 0.1, min: 0.1 },
+        },
+        type: 'fiat' as const,
+    },
+    demo_account: 0 as const,
+    displayBalance: '0.00 USD',
+    landingCompanyName: 'svg' as const,
+    loginid: '',
+    transfer: 'all',
+};
+
+const mockWalletsTransfer = {
+    accountLimits: {},
+    activeWalletExchangeRates: {},
+    formData: { fromAmount: 100, toAmount: 200 },
+    fromAccount: mockWallets,
+    toAccount: mockWallets,
+    USDExchangeRates: {},
+};
+
+describe('useTransferMessages', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    test('should render default message and type correctly for verified account', () => {
+        const { result } = renderHook(() => useTransferMessages(mockWalletsTransfer));
+
+        expect(result.current).toEqual([
+            {
+                message: {
+                    text: 'Fee: {{feeMessageText}} ({{feePercentage}}% transfer fee or {{minimumFeeText}}, whichever is higher, applies for fund transfers between your {{fiatAccountName}}{{conjunction}} cryptocurrency Wallets)',
+                    values: {
+                        conjunction: ' Wallet and ',
+                        feeMessageText: '0.1 USD',
+                        feePercentage: undefined,
+                        fiatAccountName: undefined,
+                        minimumFeeText: '0.1 USD',
+                    },
+                },
+                type: 'info',
+            },
+        ]);
+    });
```

Comment:
Hi @lubega-deriv,
I feel these are integration tests as message-functions themselves are not mocked.
We can have separate test files to test the functions.

Reply:
Updated